Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -2443,6 +2443,48 @@ theorem sup_mul_nat (o : Ordinal) : (sup fun n : ℕ => o * n) = o * ω := by
     exact sup_eq_zero_iff.2 fun n => zero_mul (n : Ordinal)
   · exact (mul_isNormal ho).apply_omega0
 
+theorem coe_succ_Iio {o : Ordinal} (h : o.IsLimit) {α : Iio o} : (succ α).1 = α.1 + 1 := by
```

Comment:
```suggestion
theorem coe_succ_Iio {o : Ordinal} (h : o.IsLimit) (x : Iio o) : (succ x).1 = succ x.1 := by
```
Can this theorem be generalized to `SuccOrder` by taking a `IsSuccLimit` hypothesis instead?

Reply:
I'll generalize this but I don't think `x` should be explicit, so far I only use this theorem in `rw`. I didn't want `h` to be explicit either but lean could never insert it.